db/schema_tables.cc

```diff
@@ -3251,10 +3251,10 @@ future<> drop_column_mapping(utils::UUID table_id, table_schema_version version)
     const static sstring DEL_COLUMN_MAPPING_QUERY =
         format("DELETE FROM system.{} WHERE cf_id = ? and schema_version = ?",
             db::schema_tables::SCYLLA_TABLE_SCHEMA_HISTORY);
-    return qctx->qp().execute_internal(
+    co_await qctx->qp().execute_internal(
```

Comment:
nitpick: should you add `(void)` to remind the reader that this returns something which is then dropped?
(I know it's not necessary, just a thought).